tests/Tests/ORM/Proxy/ProxyFactoryTest.php

```diff
@@ -239,6 +240,41 @@ public function testProxyClonesParentFields(): void
         self::assertSame(1000, $cloned->getSalary(), 'Expect properties on the CompanyEmployee class to be cloned');
         self::assertSame('Bob', $cloned->getName(), 'Expect properties on the CompanyPerson class to be cloned');
     }
+
+    public function testUnmanagedPropertyTriggerProxyLoad(): void
+    {
+        $product = new Product('someName');
+        $product->setImage('someImage');
+        $identifier = ['id' => 42];
+
+        $classMetaData = $this->emMock->getClassMetadata(Product::class);
+
+        $persister = $this
+            ->getMockBuilder(BasicEntityPersister::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->uowMock->setEntityPersister(Product::class, $persister);
+
+        $persister
+            ->expects(self::atLeastOnce())
+            ->method('loadById')
+            ->with(self::equalTo($identifier))
```

Comment:
Is the call to `self::equalTo` necessary?

```suggestion
            ->with($identifier)
```

Reply:
necessary no. but saw another test that does that (line 79), so I also do it here.